src/gettingstarted/own-code/project-configuration.md

```diff
@@ -0,0 +1,87 @@
+# Import your own code
+
+## Configuring projects
+
+In the previous step, you created a new project on Platform.sh using the CLI. Now, there are a few configuration steps left that will help Platform.sh know what to do with your application during builds and deployments.
+
+1. **Consult a template alongside this guide**
+
+    As you go through this guide, example files will be provided that will give you a good impression of how to configure applications on Platform.sh in the language of your choice. However, since they are simple examples and your own application may require more detailed configuration than those examples address, it is recommended that you take a look at our maintained templates for additional guidance.
```

Comment:
"for the language of your choice".  "in the language of your choice" implies English, Spanish, German, etc.